worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -847,12 +774,119 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
   }
 
   def getActiveShuffleSize(): Long = {
-    fileInfos.values().asScala.map(_.values().asScala.map(_.getBytesFlushed).sum).sum
+    nonMemoryFileInfos.values().asScala.map(_.values().asScala.map(_.getBytesFlushed).sum).sum
   }
 
   def getActiveShuffleFileCount(): Long = {
-    fileInfos.asScala.values.map(_.size()).sum
+    nonMemoryFileInfos.asScala.values.map(_.size()).sum
+  }
+
+  def createFile(
```

Comment:
I don't think we should extract this method. Is it for test purpose? Better to change the test code.

Reply:
I think this change could make the code look cleaner.